admin/src/pages/MailingCampaignDetail.tsx

```
@@ -0,0 +1,197 @@
+      {/* Email preview */}
+      <Card className="p-0">
+        <div className="px-4 py-3 border-b bg-gray-50">
+          <h3 className="text-sm font-medium text-gray-700">Email Content Preview</h3>
+        </div>
+        <div className="p-4">
+          <div className="text-sm text-gray-500 mb-2">
+            <strong>Subject:</strong> {campaign.subject}
+          </div>
+          <div
+            className="border rounded-lg p-4 bg-white prose prose-sm max-w-none"
+            dangerouslySetInnerHTML={{ __html: campaign.bodyHtml }}
```

Biome (2.3.14)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)